src/logger.ts

```diff
@@ -42,7 +42,9 @@ function debug(...args) {
     return log('debug', ...args);
 }
 
-async function log(level, ...args) {
+type Level = 'error' | 'warn' | 'info' | 'debug';
+
+async function log(level: Level, ...args) {
```

Comment:
`args` should be typed

Reply:
Okay, added types, though the type is `any[]`.